src/pages/RegisterFamilyUsers.vue

```diff
@@ -45,6 +45,7 @@
             class="justify-start z-1 bg-white text-primary text-center justify-content-center border-none border-round p-2 h-3rem hover:surface-300 hover:text-900 border-none"
             style="width: 8rem; margin-left: 1rem"
             @click="activeIndex = 0"
+            :loading="spinner"
```

Comment:
Should this not be the `disabled` prop instead of `loading`? The `loading` prop would usually be used on buttons that perform, for example, a submit action. However, this is just back button, right? 

```suggestion
            :disabled="spinner"
```

Reply:
Ah, I think disabled is probably the better prop. Thanks for the suggestion – I had seen Adam's comment on the loading prop and used that after it had sufficiently disabled the button. I'll commit your suggestion now. Thanks!